pkg/collector/corechecks/system/iostats_pdh_windows.go

```diff
@@ -0,0 +1,141 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2018 Datadog, Inc.
+// +build windows
+
+package system
+
+import (
+	"bytes"
+	"fmt"
+	"regexp"
+	"syscall"
+	"unsafe"
+
+	log "github.com/cihub/seelog"
+
+	"github.com/DataDog/datadog-agent/pkg/aggregator"
+	"github.com/DataDog/datadog-agent/pkg/collector/check"
+	core "github.com/DataDog/datadog-agent/pkg/collector/corechecks"
+	"github.com/DataDog/datadog-agent/pkg/util/winutil"
+	"github.com/DataDog/datadog-agent/pkg/util/winutil/pdhutil"
+)
+
+var (
+	modkernel32 = syscall.NewLazyDLL("kernel32.dll")
+
+	procGetLogicalDriveStringsW = modkernel32.NewProc("GetLogicalDriveStringsW")
+	procGetDriveType            = modkernel32.NewProc("GetDriveTypeW")
+)
+
+const (
+	ERROR_SUCCESS        = 0
+	ERROR_FILE_NOT_FOUND = 2
+	DRIVE_REMOVABLE      = 2
```

Comment:
`DRIVE_REMOVABLE` and `ERROR_FILE_NOT_FOUND` don't seem to be used. should we keep them or can we remove them? It's cool if we should keep them, just curious

Reply:
Those are the other constants for GetDriveType(), so they're there for completeness.